common/app/views/fragments/containers/facia_cards/standardHeaderMeta.scala.html

```diff
@@ -20,13 +20,11 @@
                     "tone-colour" -> isContributor))">
                     @href.map { href =>
                     <a data-link-name="section heading" href="@LinkTo {/@href}">
-                    @if(ArticleBadgesSwitch.isSwitchedOn){
                         @badgeFor(containerDefinition).map { badge =>
```

Comment:
indentation 😛

Reply:
Haha I think it was the wrong indentation before, now it makes more sense